core/store/src/trie/mod.rs

```diff
@@ -410,13 +410,15 @@ pub struct Trie {
     pub counter: TouchedNodesCounter,
 }
 
-/// Stores reference count change for some Trie key and value.
+/// Stores reference count change for some key-value pair in DB.
 #[derive(BorshSerialize, BorshDeserialize, Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
 pub struct TrieRefcountChange {
-    /// Hash of TrieKey.
-    key_hash: CryptoHash,
-    /// Value corresponding to the TrieKey stored in Trie.
-    value: Vec<u8>,
+    /// Hash of trie_node_or_value and part of the DB key.
+    /// Used for uniting with shard id to get actual DB key.
+    trie_node_or_value_hash: CryptoHash,
```

Comment:
Nothing actionable, just a drive by comment:

Not a huge fan of `x_or_y` names, they are bulky. But also I can't suggest a better alternative. In rust-analyzer, we have exactly the same problem where we often need to deal with `NodeOrToken` types, By convention, we use `element` term there (element = a interior node or a leat token), but that only works if the name is sufficiently freqently used.